packages/playwright-core/src/server/registry/index.ts

```diff
@@ -1025,6 +1025,17 @@ export class Registry {
     const lockfilePath = path.join(registryDirectory, '__dirlock');
     const linksDir = path.join(registryDirectory, '.links');
 
+    // proper-lockfile does retry EACCES errors as well hence we do a best-effort permission check here to not end up stalling.
+    try {
+      const lockfileTestPath = path.join(registryDirectory, '__dirlock-test-' + process.pid);
+      await fs.promises.writeFile(lockfileTestPath, '');
+      await fs.promises.unlink(lockfileTestPath);
+    } catch (e) {
+      if (e.code === 'EACCES' || e.code === 'EPERM') {
```

Comment:
`EPERM` is often an anti-virus error that goes away after a retry. Perhaps do this only on linux? Maybe even only inside a docker? Also, a test would be nice.